ansible/all-in-one/roles/xsnippet/templates/cert-watcher.path.j2

```diff
@@ -0,0 +1,5 @@
+[Path]
+PathModified={{ watch_path }}
```

Comment:
This is a really cool feature! I did not know systemd could do something like that :+1: